scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -156,7 +158,7 @@ internal class ScenarioServiceImpl(
       scenario: Scenario
   ): Scenario {
     // Validate organizationId
-    organizationService.findOrganizationById(organizationId)
+    //    organizationService.findOrganizationById(organizationId)
```

Comment:
I don't know if this should be commented: it verifies existence of organization I think, but next line too...